deploy/docker/localkube-dind/start.sh

```diff
@@ -19,25 +19,28 @@ mount --make-shared /
 export CNI_BRIDGE_NETWORK_OFFSET="0.0.1.0"
 /dindnet &> /dev/null &
 
-
-
-mkdir -p /etc/localkube
-base=/etc/localkube/docker
+mkdir -p /etc/localkube-docker
+base=/etc/localkube-docker/docker
 /usr/bin/dockerd \
   --exec-root=$base.exec \
   --graph=$base.graph \
   --host=unix://$base.socket \
   --pidfile=$base.pid &> /dev/null &
 
+mkdir -p /etc/localkube-docker
 
 mkdir -p /etc/kubernetes/manifests
+mkdir -p /var/log/localkube
 /localkube start \
 --apiserver-insecure-address=0.0.0.0 \
---apiserver-insecure-port=8080 \
+--apiserver-insecure-port=4321 \
 --network-plugin=cni \
+--generate-kubeconfig=true \
 --extra-config=kubelet.DockerEndpoint=unix:///$base.socket \
 --extra-config=kubelet.KubeletFlags.ContainerRuntimeOptions.CNIConfDir="/etc/cni/net.d" \
 --extra-config=kubelet.KubeletFlags.ContainerRuntimeOptions.CNIBinDir="/opt/cni/bin" \
 --extra-config=kubelet.ClusterDNS="10.96.0.10" \
 --extra-config=kubelet.ClusterDomain="cluster.local" \
---extra-config=kubelet.AllowPrivileged="true"
+--extra-config=kubelet.AllowPrivileged="true" \
+--v=100 \
+--logtostderr &> /var/log/localkube/logs.txt
```

Comment:
newline

Reply:
done.  Also added auto-newline setting to editor, hopefully this will be the last newline comment

Reply:
theres a vs-code addon if you haven't gotten it 😂